package.json

```diff
@@ -1,27 +1,58 @@
 {
-  "name": "cra-template",
+  "name": "cra-template-redux",
   "version": "1.0.0",
+  "scripts": {
+    "prettify": "prettier --trailing-comma es5 --single-quote --write './template/src/**/*.js'",
+    "lint": "eslint './template/src/**/*.js'",
+    "test": "jest"
+  },
   "keywords": [
     "react",
     "create-react-app",
-    "template"
+    "template",
+    "redux",
+    "reduxjs",
+    "react-redux"
   ],
-  "description": "The base template for Create React App.",
+  "description": "The official Redux+JS template for Create React App.",
   "main": "template.json",
   "repository": {
     "type": "git",
-    "url": "https://github.com/facebook/create-react-app.git",
-    "directory": "packages/cra-template"
+    "url": "https://github.com/reduxjs/cra-template-redux.git"
   },
   "license": "MIT",
   "engines": {
     "node": ">=8"
   },
   "bugs": {
-    "url": "https://github.com/facebook/create-react-app/issues"
+    "url": "https://github.com/reduxjs/cra-template-redux/issues"
   },
   "files": [
     "template",
     "template.json"
-  ]
+  ],
+  "devDependencies": {
```

Comment:
Where in the template process are these devDeps actually used? Why isn't most of this already handled as part of CRA / `react-scripts` itself?

Reply:
These dependencies don't end up in the project created by cra. These are just used by us to lint and test the code.